README.md

```diff
@@ -106,79 +102,71 @@ process.
 > It is designed to make programming web applications easier by making
 assumptions about what every developer needs to get started. It makes the
 assumption that there is a "best" way to do things, and it's designed to
-encourage that way - and in some cases to discourage alternatives. - Ruby on
+encourage that way - and in some cases to discourage alternatives. -- Ruby on
 Rails guide
 
-> Rails is a framework with lots of rules/conventions. Pay attention to the
+Rails is a framework with lots of rules/conventions. Pay attention to the
 conventions you'll need to follow for Rails throughout the week.
 
-### Rails Walkthrough (5 min)
+### Rails Walkthrough (5 minutes / 0:45)
 
-Let's walk through a Rails App to get comfortable with it's file structures and
+Let's walk through a Rails App to get comfortable with its file structure and
 identify where we will be configuring the all of the concepts we discussed
-above!  Today, we'll be taking a look at [Tunr](https://github.com/ga-wdi-exercises/tunr_rails_views_controllers/tree/solution)!
-(yes, use the solution branch!)
+above.
+
+Today, we'll be taking a look at [Tunr](https://github.com/ga-wdi-exercises/tunr_rails_views_controllers/tree/solution). **Make sure to check out the solution branch.**
 
 As we go through the app and code, you will notice how everything is abstracted
 into individual files and directories. Why?
 
 - Separation of concerns
 - Readability
-- Convention over configuration (rails conventions)
-
-## You Do: Explore the folders (5 min)
-
-Instructions:
+- Rails Conventions
 
-* Clone (no need to fork) [Tunr](https://github.com/ga-wdi-exercises/tunr_rails_views_controllers/tree/solution)
+## You Do: Explore the Folders (5 minutes / 0:50)
```

Comment:
you do: scavenger hunt!

find:

- schema
- layout.erb but the rails one
- where the css goes
- something else